Resources/Locale/en-US/ninja/ninja-actions.ftl

```diff
@@ -0,0 +1,24 @@
+action-name-toggle-ninja-gloves = Toggle ninja gloves
+action-desc-toggle-ninja-gloves = Toggles all glove actions on left click. Includes your doorjack, draining power, stunning enemies, downloading research and calling in a threat.
+
+action-name-toggle-phase-cloak = Phase cloak
+action-desc-toggle-phase-cloak = Toggles your suit's phase cloak. Beware that if you are hit, all abilities are disabled for 5 seconds, including your cloak!
+ninja-no-power = Not enough charge in suit battery!
+
+action-name-create-soap = Create soap
+action-desc-create-soap = Channels suit power into creating a bar of ninja soap. The future is now, old man!
+
+action-name-recall-katana = Recall katana
+action-desc-recall-katana = Teleports the Energy Katana linked to this suit to its wearer, cost based on distance.
+
+action-name-em-burst = EM Burst
+action-desc-em-burst = Disable any nearby technology with an electro-magnetic pulse.
+
+ninja-full-power = Suit battery is already full
+ninja-drain-empty = {CAPITALIZE(THE($battery))} does not have enough power to drain
+ninja-drain-success = You drain power from {THE($battery)}!
+
+ninja-download-fail = No new research nodes were copied...
+ninja-download-success = Copied {$count} new nodes from {THE($server)}.
+
+ninja-terror-already-called = You already called in a threat!
```

Comment:
```suggestion
ninja-terror-already-called = You already called in a threat!

action-name-katana-dash = 
action-desc-katana-dash = 
```

These two are missing from here which is causing the ability name and desc to be called "action-name-katana-dash, action-desc-katana-dash" Fill in the blanks for your flavor of text.

Reply:
i swear i had this in my bad